Add an article for Kyverno

ribbybibby opened the pull request:
Document policies that are relevant to Chainguard Containers.

[ ] Check if this is a typo or other quick fix and ignore the rest :)

## Type of change
<!-- Please be sure to add the appropriate label to your PR. -->

Documentation.

### What should this PR do?
<!-- Does this PR resolve an issue? Please include a reference to it. -->

Adds an article describing Kyverno policies that are relevant to Chainguard Containers.

### Why are we making this change?
<!-- What larger problem does this PR address? -->

Customers often ask me how they can enforce or track usage of Chainguard in their Kubernetes clusters. Kyverno is one of two (the other being Gatekeeper) admission controllers that are commonly already running in a customer environment or that a customer may be considering introducing.

This provides specific examples of policies that we can point customers to, which cuts out the work for customers of having to figure this out from the Kyverno docs.

### What are the acceptance criteria? 
<!-- What should be happening for this PR to be accepted? Please list criteria. -->
<!-- Do any stakeholders need to be tagged in this review? If so, please add them. -->

🤷 

### How should this PR be tested?
<!-- What should your reviewer do to test this PR? Please list steps. -->

1. Create a kind cluster or another local Kubernetes cluster.
```
kind create cluster
```
2. Install Kyverno.
```
helm repo add kyverno https://kyverno.github.io/kyverno/
helm repo update
helm install kyverno kyverno/kyverno -n kyverno --create-namespace
```
3. Follow the examples in the doc.

ribbybibby committed:
Add an article for Kyverno

Document policies that are relevant to Chainguard Containers.

SharpRake commented on content/chainguard/chainguard-images/staying-secure/enforcement/kyverno/index.md:
````diff
@@ -0,0 +1,481 @@
+
+Try to to create a pod with a Chainguard image.
+
+```shell
````
This command is the only one that didn't work correctly for me. I tried playing with the policy YAML but no matter what kyverno kept preventing me from using Chainguard images.

I had claude take a look at it and it seemed to think the issue was that Kyverno couldn't access the registry to fetch the metadata - not sure if that makes any sense

SharpRake reviewed (commented):
Testing was mostly successful! There's one command (see comment) that's not working for me but the rest works well enough I can move forward with a  close review.

SharpRake committed:
Update index.md

Signed-off-by: Mark Drake <33191761+SharpRake@users.noreply.github.com>

SharpRake committed:
Update index.md

removing third example (ensuring only CG containers are used) as I'm getting inconsistent results.

Signed-off-by: Mark Drake <33191761+SharpRake@users.noreply.github.com>

SharpRake reviewed (approved):
As discussed on Slack, I removed the third example as I was getting inconsistent results from the Kyverno policy. Otherwise, this is in great shape and is an awesome addition to Academy. Thanks @ribbybibby!
